src/AppBundle/Resources/views/base.html.twig

```diff
@@ -53,6 +53,35 @@
       window.AppData.countryIso = "{{ country_iso }}";
     </script>
     <script src="{{ asset('js/common.js') }}"></script>
+    {# Code for top cart #}
+    <script>
+    window.AppData = window.AppData || {};
+    {% set sessionCart = app.session.get('cart') %}
+    {% if sessionCart is not empty and sessionCart.restaurant %}
```

Comment:
With the `if`, it means `window.AppData.Cart` can actually be `undefined` when there is no session. 
It is breaking some code which is relying on it. This is "normal" because in the previous code, the cart was always defined. 

I think it would be more reliable to always have `window.AppData.Cart` defined, even if there is no session. 
In this case, most of the properties would be initialized to `null` (like `date`)